src/main/java/teammates/ui/webapi/GetAccountAction.java

```diff
@@ -13,12 +13,13 @@ class GetAccountAction extends AdminOnlyAction {
     public JsonResult execute() {
         String googleId = getNonNullRequestParamValue(Const.ParamsNames.INSTRUCTOR_ID);
 
-        AccountAttributes accountInfo = logic.getAccount(googleId);
-        if (accountInfo == null) {
+        Account account = sqlLogic.getAccountByGoogleId(googleId);
```

Comment:
Need to support dual db for this. Account has an isMigrated flag you need to check.

Reply:
Have made this change!